src/stripe/Components/InfoPriceColumn.js

```diff
@@ -1,15 +1,15 @@
-/**
- * Use the CSS tab above to style your Element's container.
- */
 import React from 'react';
 import { __ } from '@wordpress/i18n';
 
 const InfoPriceColumn = (props) => {
-	const { price, name } = props;
-	// TODO: Get vat from db when it will be implemented.
-	const vatPer = 22;
-	const vat = ((price / 100) * vatPer).toFixed(2);
-	const total = parseFloat(price) + parseFloat(vat);
+	const { price, name, taxPercentage, taxAmount } = props;
+	const total = parseFloat(price) + parseFloat(taxAmount);
+
+	let cutVat = false;
+	if (taxPercentage === 0) {
+		cutVat = ((price / 100) * 22).toFixed(2);
+	}
+
```

Comment:
When working with currencies, it's better to use some library already tested, like this [one](https://github.com/dinerojs/dinero.js)

Reply:
I used the value from db.